jsonlogging/pom.xml

```diff
@@ -0,0 +1,65 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.secondbase</groupId>
+        <artifactId>secondbase</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jsonlogging</artifactId>
+    <packaging>jar</packaging>
+    <name>jsonlogging</name>
+
+    <properties>
+      <joda-time.version>2.3</joda-time.version>
```

Comment:
Done. Although the idea was to put things that are used more than once in a property. I'm sure we'll now update all other pom files to follow the same pattern...